relay-server/src/services/project_cache.rs

```diff
@@ -520,36 +519,12 @@ impl GlobalConfigStatus {
 
 impl ProjectCacheBroker {
     fn set_global_config(&mut self, global_config: Arc<GlobalConfig>) {
-        let GlobalConfigStatus::Pending(project_keys) = std::mem::replace(
+        let GlobalConfigStatus::Pending = std::mem::replace(
```

Comment:
This is now just a `self.global_config = GlobalConfigStatus::Ready(global_config);`

But shouldn't we trigger an unspool here if it was pending so we don't have to wait for the backoff?